scripts/sentry-xcode-debug-files.sh

```diff
@@ -21,7 +21,7 @@ EXTRA_ARGS="$SENTRY_CLI_EXTRA_ARGS $SENTRY_CLI_DEBUG_FILES_UPLOAD_EXTRA_ARGS $IN
 UPLOAD_DEBUG_FILES="\"$SENTRY_CLI_EXECUTABLE\" debug-files upload $EXTRA_ARGS \"$DWARF_DSYM_FOLDER_PATH\""
 
 if [ "$SENTRY_DISABLE_AUTO_UPLOAD" != true ]; then
-  /bin/sh -c "$UPLOAD_DEBUG_FILES"
+  /bin/sh -c "$NODE_BINARY $UPLOAD_DEBUG_FILES"
```

Comment:
We should wrap the node binary so paths with white spaces do not cause issues. But with that, we have to make sure the `NODE_BINARY` is always defined otherwise `'' 'sentry-cli'` will fail to execute.

```suggestion
  /bin/sh -c "\"$NODE_BINARY\" $UPLOAD_DEBUG_FILES"
```